client/app.js

```diff
@@ -75,8 +76,19 @@ porybox.config(['$mdThemingProvider','$routeProvider',function(
   $routeProvider.otherwise({ redirectTo: '/' });
 }]);
 
+// Add a ?v=1.0.0 (e.g.) query to all requests for templates
+// This avoids browser cache issues when the version number is bumped
```

Comment:
Oh, nice workaround. I was worried we would have to append it to everything manually, that's much nicer.